packages/overlay/src/components/Debugger.tsx

```diff
@@ -57,30 +49,9 @@ export default function Debugger({
   isOnline: boolean;
   setTriggerButtonCount: (count: NotificationCount) => void;
   fullPage: boolean;
-  setReloadSpotlight: React.Dispatch<React.SetStateAction<number>>;
 }) {
   const isSentryIntegrationAdded = integrations.some(integration => integration.name === 'sentry');
-  const { setEvents } = useContext(SentryEventsContext);
```

Comment:
I would rather not have any traces of Electron app in Spotlight itself when not absolutely necessary. Rather let's add an option - `showClearEventsButton` to Spotlight itself, which can be default `true` and in the electron app when we init Spotlight we set it to false.

Reply:
👍🏻